extensions/test/CognitoAuthentication.IntegrationTests.Xamarin/CognitoAuthentication.IntegrationTests.Xamarin/_xamarin/BaseAuthenticationTestClass.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+
+using Amazon.Runtime;
+using Amazon.CognitoIdentityProvider;
+using Amazon.CognitoIdentity;
+using Amazon.IdentityManagement;
+
+namespace Amazon.Extensions.CognitoAuthentication.IntegrationTests
+{
+    /// <summary>
+    /// Base class to be used for authentication integrations tests
+    /// Allows for child classes to create, sign up, or confirm users
+    /// </summary>
+    public partial class BaseAuthenticationTestClass
+    {
+        protected RegionEndpoint clientRegion = Settings.RegionEndpoint; 
+
+        public AmazonCognitoIdentityProviderClient GetAmazonCognitoIdentityProviderClient()
```

Comment:
could you fix the whitespace changes?  

setting the default to convert tabs to 4 spaces should help.  See this [stackoverflow](https://stackoverflow.com/questions/14167033/visual-studio-replace-tab-with-4-spaces).

Reply:
Submitted a change to fix the issue

Reply:
I still see a lot of indentation issues esp. lines 44~55.

Reply:
there are tabs in BaseAuthenticationTestClass.cs, App.xaml.cs and MainPage.xaml.cs.  subbmiyed a change fix the issue